widgets/KinesisSource-streamingsource.json

```diff
@@ -0,0 +1,108 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "Kinesis Configuration",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Application Name",
+          "name": "appName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Stream Name",
+          "name": "streamName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Kinesis endpoint url",
+          "name": "endpointUrl"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "region",
+          "name": "region"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Checkpoint interval duration",
+          "name": "duration"
+        },
+        {
+          "widget-type": "select",
+          "label": "Initial position in stream",
+          "name": "initialPosition",
+          "widget-attributes": {
+            "values": [
+              "LATEST",
+              "TRIM_HORIZON"
+            ],
+            "default": "LATEST"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "AWS access key id",
+          "name": "awsAccessKeyId"
+        },
+        {
+          "widget-type": "password",
+          "label": "AWS access secret",
+          "name": "awsAccessSecret"
+        }
+      ]
+    },
+    {
+      "label": "Format",
+      "properties": [
+        {
+          "widget-type": "select",
+          "label": "Format",
+          "name": "format",
+          "widget-attributes": {
+            "values": [
+              "",
```

Comment:
I kept it empty string to be on the same page with kafka streaming source
https://github.com/caskdata/hydrator-plugins/blob/develop/spark-plugins/widgets/Kafka-streamingsource.json#L81

let me know if we need to change it in both the sources